packages/in_app_purchase/in_app_purchase_android/lib/src/pigeon_converters.dart

```diff
@@ -110,16 +156,104 @@ PlatformProductType platformProductTypeFromWrapper(ProductType type) {
   };
 }
 
+/// Creates a [PricingPhaseWrapper] from its Pigeon equivalent.
```

Comment:
In most Pigeon conversions this new code would be offset by removing a bunch of manual JSON conversion on the Dart side (like it is on the Java side in this PR); in this case it's net new code for now because we can't remove the generated JSON conversion yet (as noted in the PR description).